diff --git a/model-common/src/main/java/mx/com/nmp/midas/ConstantsParametros.java b/model-common/src/main/java/mx/com/nmp/midas/ConstantsParametros.java
index 4361d5f2ed..697ce79d71 100644
--- a/model-common/src/main/java/mx/com/nmp/midas/ConstantsParametros.java
+++ b/model-common/src/main/java/mx/com/nmp/midas/ConstantsParametros.java
@@ -792,4 +792,7 @@ public interface ConstantsParametros extends Serializable{
     public static final long ID_PARAM_SUCURSAL_CERTIFICADO_NO_EXHIBICION_CAJERO = 542L;
     public static final long ID_PARAM_SUCURSAL_CERTIFICADO_NO_EXHIBICION_SUCURSAL = 543L;
 
+    //MIPP
+    public static final long ID_PARAM_MOSTRAR_NIVEL_CLIENTE  = 565L;
+
 }
diff --git a/web/src/main/java/mx/com/nmp/midas/webapp/action/ClienteForm.java b/web/src/main/java/mx/com/nmp/midas/webapp/action/ClienteForm.java
index cea49dd421..9691b2cdd2 100644
--- a/web/src/main/java/mx/com/nmp/midas/webapp/action/ClienteForm.java
+++ b/web/src/main/java/mx/com/nmp/midas/webapp/action/ClienteForm.java
@@ -173,6 +173,10 @@ public class ClienteForm extends BaseAction{
     //Backlog: TASK0211819
     private Date clienteExpedienteFecha;
 
+	private boolean mostrarNivelClienteSucursal;
+
+	private UsuarioDTO usuario;
+
 	public ClienteForm() {
 		super();
 		listaCapturasAnteriores = new ArrayList<ClienteViewBean>(TAMANIO_LISTA_CAPTURAS_ANTERIORES);
@@ -206,13 +210,14 @@ public class ClienteForm extends BaseAction{
 					}
 				});
 
-			UsuarioDTO usuario = (UsuarioDTO) getSessionAttribute(ConstantsView.USUARIO_SESION);
+			usuario = (UsuarioDTO) getSessionAttribute(ConstantsView.USUARIO_SESION);
 			ParametroSucursal parametroProfecoHabilitado = parametroSucursalEjb.obtenerParametro(usuario.getIdSucursal(), ConstantsParametros.ID_PARAM_SUCURSAL_PROFECO_HABILITADO);
 			profecoHabilitado = Math.round(parametroProfecoHabilitado.getValor()) == 1F;
 			idBeneficiarioSeleccionado = -1;
 
 			campaniasActivas = new ArrayList<CampaniaOperacionDTO>();
 			canalesActivos = new ArrayList<CanalOrigenDTO>();
+			mostrarNivelClienteSucursal = Boolean.FALSE;
 
 		} catch (ServiceException e) {
 			exceptionHandler.handle(e);
@@ -517,6 +522,7 @@ public class ClienteForm extends BaseAction{
 	}
 
 	public void obtenerCliente(){
+		ParametroSucursal mostrarNivelCliente;
 		PrendaValuadorCapturaBilleteUnico prenda = null;
 		prenda = (PrendaValuadorCapturaBilleteUnico)getSession().getAttribute("billeteUnico");
 		if (prenda!=null){
@@ -554,8 +560,13 @@ public class ClienteForm extends BaseAction{
 
 			if (cliente.getNumRegistro() > 0) {
 				try {
-				NivelClienteDTO nivelCliente = clienteManager.getNivelClienteByNumRegistro(cliente);
-				nivelClienteActual = nivelCliente.getNivelClienteActual();
+					mostrarNivelCliente = parametroSucursalEjb.obtenerParametro(usuario.getIdSucursal(), ConstantsParametros.ID_PARAM_MOSTRAR_NIVEL_CLIENTE);
+					if(mostrarNivelCliente != null && mostrarNivelCliente.getValor() > 0){
+						mostrarNivelClienteSucursal = true;
+					}
+
+					NivelClienteDTO nivelCliente = clienteManager.getNivelClienteByNumRegistro(cliente);
+				    nivelClienteActual = nivelCliente.getNivelClienteActual();
 				} catch (ServiceException e) {
 					//exceptionHandler.handle(e);
 					addMessage(MSG_ERROR_NIVEL_CLIENTE);
@@ -1769,4 +1780,9 @@ public class ClienteForm extends BaseAction{
     public Date getClienteExpedienteFecha() {
         return clienteExpedienteFecha;
     }
+
+	public boolean isMostrarNivelClienteSucursal() {
+		return mostrarNivelClienteSucursal;
+	}
+
 }
diff --git a/web/src/main/webapp/clienteDetalle.xhtml b/web/src/main/webapp/clienteDetalle.xhtml
index f7e9cd0414..2191288bf1 100644
--- a/web/src/main/webapp/clienteDetalle.xhtml
+++ b/web/src/main/webapp/clienteDetalle.xhtml
@@ -167,16 +167,16 @@
 				</m:disablePanelGroup>
 			</h:panelGroup>
 			
-			<h:panelGroup id="panelNivelCliente">
+			<h:panelGroup id="panelNivelCliente" rendered="#{clienteForm.mostrarNivelClienteSucursal}">
 					<fieldset>
 						<legend>#{text['cliente.lgn.nivel.cliente']}</legend>
 						<table>
 							<tr>
 								<td><h:outputText id="lblNivelCliente"
-										value="#{text['cliente.lbl.nivel.cliente']}" rendered="true" /></td>
+										value="#{text['cliente.lbl.nivel.cliente']}" rendered="#{clienteForm.mostrarNivelClienteSucursal}" /></td>
 								<td><h:outputText id="nivelCliente"
 										title="#{text['cliente.lbl.nivel.cliente']}"
-										value="#{clienteForm.nivelClienteActual}" rendered="true" /></td>
+										value="#{clienteForm.nivelClienteActual}" rendered="#{clienteForm.mostrarNivelClienteSucursal}" /></td>
 							</tr>
 						</table>
 					</fieldset>
diff --git a/web/src/main/webapp/clienteDetalleSimuladorEmpenio.xhtml b/web/src/main/webapp/clienteDetalleSimuladorEmpenio.xhtml
index b73dea46dc..950a38be92 100644
--- a/web/src/main/webapp/clienteDetalleSimuladorEmpenio.xhtml
+++ b/web/src/main/webapp/clienteDetalleSimuladorEmpenio.xhtml
@@ -72,10 +72,10 @@
 						</tr>
 						<tr>
 							<td><h:outputText id="lblNivelCliente"
-											  value="#{text['cliente.lbl.nivel.cliente']}" rendered="true" /></td>
+											  value="#{text['cliente.lbl.nivel.cliente']}" rendered="#{clienteForm.mostrarNivelClienteSucursal}" /></td>
 							<td><h:outputText id="nivelCliente"
 											  title="#{text['cliente.lbl.nivel.cliente']}"
-											  value="#{clienteForm.nivelClienteActual}" rendered="true" /></td>
+											  value="#{clienteForm.nivelClienteActual}" rendered="#{clienteForm.mostrarNivelClienteSucursal}" /></td>
 						</tr>
 						<tr>
 							<td>
diff --git a/web/src/main/webapp/clienteFiltroBusqueda.xhtml b/web/src/main/webapp/clienteFiltroBusqueda.xhtml
index bb0eaa8b9e..cbfeaa6453 100644
--- a/web/src/main/webapp/clienteFiltroBusqueda.xhtml
+++ b/web/src/main/webapp/clienteFiltroBusqueda.xhtml
@@ -27,7 +27,7 @@
                             <c:if test="#{empty mostrarBotonBuscar || mostrarBotonBuscar == true}">
                             	<c:if test="#{empty overrideBusquedaBean}">
 							<a4j:commandButton ignoreDupResponses="true" id="btnObtenerCliente" value="#{text['button.search']}" action="#{clienteForm.obtenerCliente}"
-								reRender="pgModalAlertaCliente,pgJSCalculoAvaluo,datosAvaluo,numeroCredencial,panelCapturaCliente,panelDomicilioCliente,panelNivelCliente,panelIdenficacion,panelGroupEmpresa,panelGroupCotitular,panelGroupCampania,panelBotones,modalClienteBoletinado,pgScriptClienteBoletinado,pgScriptCapturaNombreCompleto,pnlVerificarHuella, pnlAutorizarGerente,modalAlertaCliente,generarContrato,generarTalonesAmarre,imprimirReferencias"
+								reRender="pgModalAlertaCliente,pgJSCalculoAvaluo,datosAvaluo,numeroCredencial,panelCapturaCliente,panelDomicilioCliente,panelNivelCliente,panelIdenficacion,panelGroupEmpresa,panelGroupCotitular,panelGroupCampania,panelBotones,modalClienteBoletinado,pgScriptClienteBoletinado,pgScriptCapturaNombreCompleto,pnlVerificarHuella, pnlAutorizarGerente,modalAlertaCliente,generarContrato,generarTalonesAmarre,imprimirReferencias,lblNivelCliente,nivelCliente"
 								oncomplete="
 										if (#{!clienteForm.verificarHuellaClienteForm.mostrarVerificarHuella}){
 											if (#{clienteForm.cliente.boletinado}){
@@ -71,7 +71,7 @@
 
 							<c:if test="#{empty mostrarBotonNuevo || mostrarBotonNuevo == true}">
 								<a4j:commandButton ignoreDupResponses="true" id="btnNuevo" value="#{text['cliente.button.new']}" action="#{clienteForm.limpiarCampos}"
-									reRender="pgJSCalculoAvaluo,datosAvaluo,numeroCredencial,panelEtiquetaNumEncontrada,panelNumeroEncontrado,panelCapturaCliente,panelDomicilioCliente,panelNivelCliente,panelIdenficacion,panelGroupEmpresa,panelGroupCotitular,panelGroupCampania,modalAutorizacionGerentee,pgScriptCapturaNombreCompleto,pgScriptClienteBoletinado,pgNuevaPartida,panelGroupAutomovil,btnCompraEmpenio,generarContrato,generarTalonesAmarre,imprimirReferencias" ajaxSingle="true"
+									reRender="pgJSCalculoAvaluo,datosAvaluo,numeroCredencial,panelEtiquetaNumEncontrada,panelNumeroEncontrado,panelCapturaCliente,panelDomicilioCliente,panelNivelCliente,panelIdenficacion,panelGroupEmpresa,panelGroupCotitular,panelGroupCampania,modalAutorizacionGerentee,pgScriptCapturaNombreCompleto,pgScriptClienteBoletinado,pgNuevaPartida,panelGroupAutomovil,btnCompraEmpenio,generarContrato,generarTalonesAmarre,imprimirReferencias,lblNivelCliente,nivelCliente" ajaxSingle="true"
 								oncomplete="navigatorEngine.setAfterResetFocus('#{ifClienteNuevo}'); navigatorEngine.resetRoot(null);limpiarTotales();"
 									disabled="#{clienteForm.clienteDeshabilitado}" />
 							</c:if>
@@ -84,7 +84,7 @@
                                                navigatorEngine.resetRootForRichModalId('modalAutorizacionGerente')}"
                                                action="#{clienteForm.limpiarCampos}"
                                                style="display:none"
-                                               reRender="pgJSCalculoAvaluo,datosAvaluo,numeroCredencial,panelEtiquetaNumEncontrada,panelNumeroEncontrado,panelCapturaCliente,panelDomicilioCliente,panelNivelCliente,panelIdenficacion,panelGroupEmpresa,panelGroupCotitular,pgScriptCapturaNombreCompleto,pgScriptClienteBoletinado,btnCompraEmpenio"
+                                               reRender="pgJSCalculoAvaluo,datosAvaluo,numeroCredencial,panelEtiquetaNumEncontrada,panelNumeroEncontrado,panelCapturaCliente,panelDomicilioCliente,panelNivelCliente,panelIdenficacion,panelGroupEmpresa,panelGroupCotitular,pgScriptCapturaNombreCompleto,pgScriptClienteBoletinado,btnCompraEmpenio,lblNivelCliente,nivelCliente"
                                                disabled="#{!clienteForm.deshabilitarCampos}" immediate="true"
                                                rendered="#{clienteForm.mostrarBotonAutorizacionNuevoCliente}"/>
 						</td>
@@ -167,7 +167,7 @@
                                        action="#{clienteForm.autorizarNuevoCliente}"
                                        oncomplete="javascript:if(#{empty facesContext.maximumSeverity}){Richfaces.hideModalPanel('modalAutorizacionGerente');}
                                        navigatorEngine.setAfterResetFocus('capturaAvaluo:nombre');navigatorEngine.resetRoot(null);"
-                                       reRender="inputClaveGerente,inputJustificacion,numeroCredencial,panelCapturaCliente,panelDomicilioCliente,panelNivelCliente,panelIdenficacion,panelGroupEmpresa,panelGroupCotitular,panelBotones,modalClienteBoletinado,pgScriptClienteBoletinado,pgScriptCapturaNombreCompleto" />
+                                       reRender="inputClaveGerente,inputJustificacion,numeroCredencial,panelCapturaCliente,panelDomicilioCliente,panelNivelCliente,panelIdenficacion,panelGroupEmpresa,panelGroupCotitular,panelBotones,modalClienteBoletinado,pgScriptClienteBoletinado,pgScriptCapturaNombreCompleto,lblNivelCliente,nivelCliente" />
                     <h:commandButton value="#{text['button.cancel']}"
                                      onclick="javascript:Richfaces.hideModalPanel('modalAutorizacionGerente');return false;" />
                 </m:modalButtons>
